metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatSaveTablePartitionMetadataOnlyPreEvent.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.common.server.events;
+
+import com.netflix.metacat.common.MetacatRequestContext;
+import com.netflix.metacat.common.QualifiedName;
+import com.netflix.metacat.common.dto.PartitionsSaveRequestDto;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Metacat save table partition metadata only pre event.
+ * @author zhenl
+ * @since 1.2.0
+ */
+@Getter
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class MetacatSaveTablePartitionMetadataOnlyPreEvent extends MetacatSaveTablePartitionPreEvent {
```

Comment:
When extending MetacatSaveTablePartitionPreEvent, will the listeners listening on MetacatSaveTablePartitionPreEvent get called for a MetacatSaveTablePartitionMetadataOnlyPreEvent?

Reply:
Will change to separate them out.